### PR TITLE
docs: add cf page back

### LINKED ISSUE #1648: Is Cloudflare Pages still active
I always used https://jellyfin-vue.pages.dev but in the last commit, I can not find the cloudflare pages build status? Did you disable cloudflare pages? If yes, why? So I did not have to deploy it on my own or do you now manually somewhere else?

### PR DESCRIPTION
**Pull request by ThibaultNocchi**
fixes #1648

### CODE REVIEW

**Comment by ferferga** [comment]
Now that you're on it, would you mind removing the Husky line and replacing it with a text that just refers to the conventional commit guidelines? (As we don't have Commitzen or Husky anymore)

**Comment by ThibaultNocchi** [comment]
Done :)

File: README.md [hunk 1]
```diff
@@ -72,17 +76,17 @@ This is an experimental web client for Jellyfin based on Vue.js. We welcome all
...
 
-   ```bash
-   npm run build
-   ```
+```bash
+npm run build
+```
 
-   Build output will be available under the `src/dist` folder.
+Build output will be available under the `src/dist` folder.
```
Comment (ferferga) [inline README.md:89]: Is the removal of the tabulation intended?

Comment (ThibaultNocchi) [reply README.md:89]: yes that's the markdown linter that did it. And it seems to make sense to me

Comment (ThibaultNocchi) [reply README.md:89]: I also did it in a specific commit :)

Comment (ferferga) [reply README.md:89]: @ThibaultNocchi How we can keep this layout then?

**master**
image

**Your branch**
image

Imo it makes sense to keep the "Build for production" part inside "Getting Started" as it's the bare minimum an user needs to build their own client version. The rest of the sections cover more "advanced" use cases and are good as they're now, but imo, these two initial sections should be kept together.

**Pull request merged**